src/datanode/src/instance/sql.rs

```diff
@@ -66,7 +66,10 @@ impl Instance {
                 )?;
                 self.sql_handler.execute(request, query_ctx).await
             }
-
+            QueryStatement::Sql(Statement::Delete(d)) => {
+                let request = SqlRequest::Delete(*d.clone());
```

Comment:
Why do we need to deref the cloned struct? Could we just move it?
```suggestion
                let request = SqlRequest::Delete(d);
```

Reply:
maybe we can use `let request = SqlRequest::Delete(*d);`, because the parameter is not of box type.